plugins/participants.js

```diff
@@ -0,0 +1,167 @@
+module.exports = (controller) => {
+    const renderParticipantsBlocks = async (participants) => {
+        return Promise.all(
+            participants.map((participant) =>
+                renderParticipantBlocks(participant)
+            )
+        ).then((participants) => participants.flat());
+    };
+
+    const renderParticipantBlocks = async (participant) => {
```

Comment:
does this function need to be marked as `async`?  I don't see the `await` keyword being used in its body

Reply:
👍